miniapps/electromagnetics/CMakeLists.txt

```diff
@@ -34,6 +34,21 @@ if (MFEM_USE_MPI)
    EXTRA_HEADERS maxwell_solver.hpp ${MFEM_MINIAPPS_COMMON_HEADERS}
    LIBRARIES mfem mfem_miniapps_common)
 
+  # Tag ALL_EXE_SRCS files to be compiled with nvcc.
+  if (MFEM_USE_CUDA)
+     set_property(SOURCE tesla.cpp PROPERTY LANGUAGE CUDA)
+     set_property(SOURCE tesla_solver.cpp PROPERTY LANGUAGE CUDA)
+     
+     set_property(SOURCE volta.cpp PROPERTY LANGUAGE CUDA)
+     set_property(SOURCE volta_solver.cpp PROPERTY LANGUAGE CUDA)
+     
+     set_property(SOURCE joule.cpp PROPERTY LANGUAGE CUDA)
+     set_property(SOURCE joule_solver.cpp PROPERTY LANGUAGE CUDA)
+     
+     set_property(SOURCE maxwell.cpp PROPERTY LANGUAGE CUDA)
+     set_property(SOURCE maxwell_solver.cpp PROPERTY LANGUAGE CUDA)
+  endif()
+   
```

Comment:
Why this change?

Reply:
To allow `ctests` to run on all tests, examples and miniapps, in `cuda` mode.